packages/utils/src/cache.ts

```diff
@@ -0,0 +1,7 @@
+export { CacheLocalStorage } from './cache/cache-localstorage.js';
+export { CacheIndexedDBStorage } from './cache/cache-indexeddb.js';
+export { CacheMap } from './cache/types.js';
+export { LocalCache, LocalCacheConfig } from './cache/local-cache.js';
+
+export { CacheStorage } from './cache/interfaces/CacheStorage.js';
+export { CacheItem } from './cache/interfaces/CacheItem.js';
```

Comment:
I think files structure can be improve. Why isn't `CacheMap` place in the same folder as `CacheStorage` and `CacheItem` ?

Reply:
Can storage be group ? Can we remove `cache-` prefix ?

- storages
       - idb
       - localstorage

Reply:
Updated by moving file. Remove prefix `cache-` and create storage folder.

Reply:
Why is `CacheMap` in types and `CacheItem`, `CacheStorage` in interfaces ? They are related and should be group together.

Reply:
Updated by moving CacheMap to interfaces folder.